BUILD.bazel

```diff
@@ -166,6 +176,24 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "wasmx_static_mod_flag",
+    flag_values = {
+        ":wasmx": "true",
+        ":wasmx_module": "static",
+    },
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "wasmx_dynamic_mod_flag",
```

Comment:
I see you renamed the wasm runtime settings to not have a `_flag` - which I think makes sense since the "setting" is more like a "state" (ie, when all of `flag_values` is set, we are in that state) rather than a flag. Wdyt about applying the same logic over here?

Reply:
That would be more consistent and I like consistency 😄